db/schema.rb

```diff
@@ -10,7 +10,35 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 2021_05_05_220831) do
+ActiveRecord::Schema.define(version: 2021_05_07_230054) do
+
+  create_table "active_storage_attachments", charset: "utf8mb4", collation: "utf8mb4_unicode_ci", force: :cascade do |t|
```

Comment:
Was this accidentally added?

Reply:
Good spot. Yes. It's from the profiles PR I was working on.